pkg/common/config/config.go

```diff
@@ -329,6 +333,9 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 	if cfg.Global.CnsRegisterVolumesCleanupIntervalInMin == 0 {
 		cfg.Global.CnsRegisterVolumesCleanupIntervalInMin = DefaultCnsRegisterVolumesCleanupIntervalInMin
 	}
+	if cfg.Global.VolumeMigrationCRCleanupIntervalInMin == 0 {
+		cfg.Global.VolumeMigrationCRCleanupIntervalInMin = DefaultVolumeMigrationCRCleanupIntervalInHours
```

Comment:
change DefaultVolumeMigrationCRCleanupIntervalInHours to DefaultVolumeMigrationCRCleanupIntervalInMinutes